pyproject.toml

```
@@ -94,7 +97,7 @@ skip = ["pp*-*"]
 test-extras = "test"
 test-command = "python -m pytest {package}/tests"
 # When cpython 3.7 is dropped we can ignore i686 (numpy not built)
-test-skip = ["pp*-*", "*-musllinux_*", "cp3{10,11,12}-win32", "cp3{8,9,10,11,12}-manylinux_i686"]
+test-skip = ["pp*-*", "*-musllinux_*", "cp3{10,11,12}-win32", "cp3{8,9,10,11,12}-manylinux_i686", "*cp313t*"]
```

```suggestion
test-skip = ["pp*-*", "*-musllinux_*", "cp3{10,11,12}-win32", "cp3{8,9,10,11,12}-manylinux_i686", "*cp313t*"]
```

This is skipping the tests on 3.13t, so they aren't currently running. Ahh, you are running the normal tests in free-threading; probably makes sense until that is fixed.